lib/index.js

```diff
@@ -19,18 +19,21 @@ function spawnSeal(){
   }
 }
 
+function gravity (){
+  narwhal.y+=.5
+}
+
 function moveInput() {
   window.addEventListener('keydown', function(e) {
-    if (e.keyCode === 37) {
-      narwhal.x -= 5;
-    } else if (e.keyCode === 38) {
-      narwhal.y -=5;
-    } else if (e.keyCode === 40) {
-      narwhal.y += 5;
-    } else if (e.keyCode === 39) {
-      narwhal.x += 5;
+    if (e.keyCode === 37) { //LEFT
+      narwhal.narwhalRotate(-5)
+    // } else if (e.keyCode === 38) {//UP
+    //   narwhal.y -=5;
+    // } else if (e.keyCode === 40) {//DOWN
+    //   narwhal.y += 5;
+    } else if (e.keyCode === 39) {//RIGHT
+      narwhal.narwhalRotate(5)
     }
```

Comment:
Adjusted the keyboard input to set a rotational degree instead of moving left and right